microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,50 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `prerelease package`.  
 
+## 1.0.674-prerelease  
+
+Release Date: Oct 19, 2020  
+
+[NuGet package][NuGetGallery1.0.674-prerelease] \| minimum Microsoft Edge version 88.0.674.0.
+
+#### General
+
+*   Added [WebView2 Group Policies][DeployedgeMicrosoftEdgeWebviewPolicies]. For more information on best practices, navigate to [group policies for WebView2][Webview2ConceptsEnterpriseGroupPoliciesForWebview2].
+*   Added [NavigateWithWebResourceRequest][ReferenceWin32Icorewebview2experimentalNavigatewithwebresourcerequest10674] method that lets you provide post data or additional request headers during navigation.
+*   Added [DOMContentLoaded][ReferenceWin32Icorewebview2experimentalAddDomcontentloaded10674] event that runs when the initial HTML document is parsed.
+*   Added the [Environment][ReferenceWin32Icorewebview2experimentalGetEnvironment10674] property on WebView2. This property exposes the WebView2 environment where an instance of WebView2 was created.
+*   Added [cookie management][ReferenceWin32Icorewebview2experimentalGetCookiemanager10674] APIs that allow developers to authenticate the WebView2 session, or retrieve cookies from WebView to authenticate other tools. We're planning to make language/framework specific improvements. For more information, navigate to [API Review: Cookie Management][GithubMicrosoftedgeWebview2AnnouncementIssue2].
+*   Updated the [WebResourceResponseReceived][ReferenceWin32Icorewebview2experimentalAddWebresourceresponsereceived10674] event, and added immutable [WebResourceResponseView][ReferenceWin32Icorewebview2experimentalwebresourceresponseview10674] and [WebResourceResponseReceivedEventArgs::PopulateResponseContent][ReferenceWin32Icorewebview2experimentalwebresourceresponsereceivedeventargsPopulateresponsecontent09628] to become [WebResourceResponseView::GetContent][ReferenceWin32Icorewebview2experimentalwebresourceresponseviewGetcontent10674].
+*   Turned off [Microsoft Defender Application Guard (WDAG)][WindowsSecurityThreatProtectionMicrosoftDefenderApplicationGuardWindows10] in WebView2.
+*   Added [SystemCursorId][ReferenceWin32Icorewebview2experimentalcompositioncontroller2GetSystemcursorid10674] for Visual Hosting.
+*   Input Method Header is now supported in visual hosting.
```

Comment:
> *   Input Method Header is now supported in visual hosting.

I think you're referring to the IME bugs we fixed? I'd just say something like `Added bug fixed for Input Method in Visual Hosting`.